zerver/models.py

```diff
@@ -1812,3 +1819,23 @@ def get_bot_services(user_profile_id):
 def get_service_profile(email, realm, service_name):
     # type: (str, Realm, str) -> Service
     return Service.objects.get(user_profile__email=email, user_profile__realm=realm, name=service_name)
+
+@cache_with_key(slash_commands_by_realm_key, timeout=3600 * 24)
+def get_slash_commands_by_realm(realm, is_active=None):
+    # type: (Realm, Optional[bool]) -> List[Any]
+    if is_active is None:
+        users = list(UserProfile.objects.filter(realm=realm, bot_type=UserProfile.SLASH_COMMANDS).values('short_name'))
+    else:
+        users = list(UserProfile.objects.filter(realm=realm, bot_type=UserProfile.SLASH_COMMANDS,
+                                                is_active=is_active).values('short_name', 'is_active'))
```

Comment:
What's the point of retrieving the `is_active` field (it looks like you don't use it below)?